projects/packages/my-jetpack/_inc/hooks/use-fullscreen/index.tsx

```diff
@@ -0,0 +1,30 @@
+import { useEffect } from 'react';
+
+/**
+ * Goes into fullscreen mode when the component is loaded
+ *
+ * @param {string[] | undefined} classes - classes to add to document.body
+ */
+export const useFullScreen = ( classes?: string[] ) => {
```

Comment:
If we don't have any use case for classes right now, let's not add this functionality. It can always be added when actually needed.